sqflite/lib/src/sentry_database.dart

```diff
@@ -32,7 +34,10 @@ class SentryDatabase extends SentryDatabaseExecutor implements Database {
   // ignore: public_member_api_docs
   static const dbSqlQueryOp = 'db.sql.query';
 
-  static const _dbSqlOp = 'db.sql.transaction';
+  static const _dbSqlTransactionOp = 'db.sql.transaction';
+
+  static const _dbSqlReadTransactionOp = 'db.sql.readTransaction';
```

Comment:
not sure if camel case is right here, from what I can read [here](https://develop.sentry.dev/sdk/performance/span-operations/) it should be snake case `db.sql.read_transaction`